quiz-category/quiz-category.html

```diff
@@ -0,0 +1,90 @@
+                <li>
+                    <a href="#" class="nav-item">Category</a>
+                    <p class="active"></p>
+                </li>
+                <li>
+                    <a href="#" class="nav-item">About us</a>
+                    <p class="active"></p>
+                </li>
+            </section>
+        </nav>
+    </header>
+    <main>
+        <div class="category-container">
+            <h2 class="category-title">Math Trivia</h2>
+            <div class="category-card-container">
+                <div class="card-container">
+                    <img src="../assets/math/circles.jpg" alt="circles" class="card-img">
+                    <div class="category-name">CIRCLE QUIZ</div>
+                </div>
+                <div class="card-container">
+                    <img src="../assets/math/trigonometry.jpg" alt="trigonometry" class="card-img">
+                    <div class="category-name">TRIGONOMETRY QUIZ</div>
+                </div>
+                <div class="card-container">
+                    <img src="../assets/math/integration.jpg" alt="integration" class="card-img">
+                    <div class="category-name">INTEGRATION QUIZ</div>
+                </div>
+            </div>
+        </div>
+        <hr>
+
+        <div class="category-container">
+            <h2 class="category-title">Hollywood Movies Trivia</h2>
+            <div class="category-card-container">
+                <div class="card-container">
+                    <img src="../assets/movies/matrix.jpg" alt="matrix" class="card-img">
+                    <div class="category-name">MATRIX QUIZ</div>
+                </div>
+                <div class="card-container">
+                    <img src="../assets/movies/harry potter.jpg" alt="harry-potter" class="card-img">
+                    <div class="category-name">HARRY POTTER</div>
+                </div>
+                <div class="card-container">
+                    <img src="../assets/movies/avengers.png" alt="avengers-endgame" class="card-img">
+                    <div class="category-name">AVENGERS: ENDGAME QUIZ</div>
+                </div>
+            </div>
+        </div>
+        <hr>
+
+        <div class="category-container">
+            <h2 class="category-title">Anime Trivia</h2>
+            <div class="category-card-container">
+                <div class="card-container">
+                    <img src="../assets/anime/one-piece.jpg" alt="math-trivia" class="card-img">
+                    <div class="category-name">ONE PIECE QUIZ</div>
+                </div>
+                <div class="card-container">
+                    <img src="../assets/anime/death-note.jpg" alt="math-trivia" class="card-img">
+                    <div class="category-name">DEATHNOTE QUIZ</div>
+                </div>
+                <div class="card-container">
+                    <img src="../assets/anime/naruto.jpg" alt="math-trivia" class="card-img">
```

Comment:
I think you can use relative import like this "/something" instead of this "../something"

Reply:
> I think you can use relative import like this "/something" instead of this "../something"

This was giving me issue in component library so I am not using it since then.